apps/kyb-app/src/pages/CollectionFlow/components/pages/CompletedScreen/CompletedScreen.tsx

```diff
@@ -3,16 +3,24 @@ import { useTranslation } from 'react-i18next';
 
 import { useCustomer } from '@/components/providers/CustomerProvider';
 import { useAppExit } from '@/hooks/useAppExit/useAppExit';
+import { useFlowTracking } from '@/hooks/useFlowTracking';
+import { CollectionFlowEventsEnum } from '@/hooks/useFlowTracking/enums';
 import { withSessionProtected } from '@/hooks/useSessionQuery/hocs/withSessionProtected';
 import { useUIOptionsRedirect } from '@/hooks/useUIOptionsRedirect';
 import { Button, Card } from '@ballerine/ui';
+import { useEffect } from 'react';
 
-export const Success = withSessionProtected(() => {
+export const CompletedScreen = withSessionProtected(() => {
   const { t } = useTranslation();
   const { customer } = useCustomer();
+  const { trackEvent } = useFlowTracking();
 
   const { exit, isExitAvailable } = useAppExit();
 
+  useEffect(() => {
+    trackEvent(CollectionFlowEventsEnum.FLOW_COMPLETED);
+  }, [trackEvent]);
```

Comment:
Shouldn't it be an empty array?

Reply:
Satisfying linter here, it wont be fired again unless prefix is changed which is dependency of `trackEvent` method.